rviz_common/src/rviz_common/loading_dialog.cpp

```diff
@@ -53,7 +53,6 @@ void LoadingDialog::showMessage(const QString & message)
   label_->setText(message);
   QApplication::processEvents();
   QWidget::repaint();
-  QApplication::flush();
```

Comment:
Following through the [QT websites](https://doc.qt.io/qt-5/qabstracteventdispatcher-obsolete.html#flush), the replacement might be `QApplication::sendPostedEvents();`

Reply:
Yeah, but I wonder if we can blindly call that, since it said it may not call that method.

Reply:
@jacobperron and I looked at the loading screen and the text appears anyways (on Linux). I also spot checked it on macOS and it was working. I think we leave it out for now.